specification/serialconsole/resource-manager/Microsoft.SerialConsole/preview/2018-07-01-preview/serialconsole.json

```diff
@@ -0,0 +1,272 @@
+                    "$ref": "./examples/DisableConsoleExample.json"
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "SerialConsoleGetResult": {
+            "title": "Serial Console GET Result",
+            "description": "Returns whether or not Serial Console is disabled.",
+            "properties": {
+                "value": {
+                    "description": "Whether or not Serial Console is disabled.",
+                    "type": "boolean"
+                }
+            }
+        },
+        "GetDisabledResult": {
+            "title": "Get Disabled result",
+            "description": "Returns whether or not Serial Console is disabled for ths given subscription",
+            "properties": {
+                "disabled": {
+                    "description": "Disabled result",
+                    "type": "boolean"
+                }
+            }
+        },
+        "SetDisabledResult": {
+            "title": "Set Disabled result",
+            "description": "Sets disable status and returns whether or not Serial Console is now disabled for ths given subscription",
+            "properties": {
+                "disabled": {
+                    "description": "Disabled result",
+                    "type": "boolean"
+                }
+            }
+        },
+        "DeploymentValidateResult": {
+            "title": "Details on Failure",
+            "description": "Details on Failure",
+            "properties": {
+                "errorMessage": {
+                    "description": "Error message",
+                    "type": "string"
+                }
+            }
+        },
+        "SerialConsoleOperations": {
+            "title": "Serial Console operations",
+            "description": "Serial Console operations",
+            "type": "object",
+            "properties": {
+                "value": {
+                    "description": "Serial Console operations",
+                    "type": "string"
+                }
+            }
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the target subscription."
+        },
+        "ApiVersionParameter": {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The API version to use for this operation."
+        },
+        "default" :{
+            "name": "default",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Default string modeled as parameter for URL to work correctly.",
+            "enum": [
+                "default"
+            ],
+            "x-ms-parameter-location": "client"
```

Comment:
Are you sure it has to be located in a client?

Reply:
My understanding is this is something the client must pass in as a parameter - hence explicitly stating the client. Did I misuse this?

Reply:
@asinn826 http://azure.github.io/autorest/extensions/#x-ms-parameter-location